<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonEditor</class>
 <widget class="QWidget" name="ButtonEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>按压</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>音效</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="normalLine"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="enterLine"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>正常</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="soundLine"/>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="soundBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>进入</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QSpinBox" name="H">
       <property name="maximum">
        <number>1999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="W">
       <property name="maximum">
        <number>1999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>大小</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="enterBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QSpinBox" name="Y">
       <property name="maximum">
        <number>1999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="X">
       <property name="maximum">
        <number>1999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>位置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="pressLine"/>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="pressBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="normalBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
